Aardvark.podspec

```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '12.0'
 
   s.source_files = 'Sources/Aardvark/**/*.{h,m,swift}'
+  s.resource_bundle = {'Aardvark' => ['Sources/Aardvark/PrivacyInfo.xcprivacy']}
```

Comment:
This is to ensure the new privacy manifest file gets picked up by Cocoapods. Relevant discussion starting [here](https://github.com/CocoaPods/CocoaPods/issues/10325#issuecomment-1718723762) and example from another pod doing the same thing [here](https://github.com/SDWebImage/SDWebImage/pull/3649/files#diff-075550cccdf1f8c42066ad20513c2beea88f70a0db3d738899d38f1f6dc742dfR47)

Reply:
Yep I used the [ListableUI podspec](https://github.com/square/Listable/blob/main/ListableUI.podspec#L17) as an example, which gets propagated [here](https://github.com/squareup/ios-cocoapodspecs/blob/master/ListableUI/9.0.0/ListableUI.podspec.json#L21-L23) in our podspecs list. And I was able to see the files references properly imported in Xcode (they're localizable strings in this example).